swap_meet/vendor.py

```diff
@@ -1,2 +1,60 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
+        if inventory:
+            self.inventory = inventory
+        else:
+            self.inventory = []
+    
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+    
+    def remove(self, item):
+        try:
+            self.inventory.remove(item)
+        except ValueError:
+            print("That item is not in the inventory.")
+            return False
+        else:
+            return item
```

Comment:
Great use of try/except here! We could restructure it a little bit to get rid of the else:
```suggestion
        try:
            self.inventory.remove(item)
            return item
        except ValueError:
            print("That item is not in the inventory.")
            return False
```